### PR TITLE
Omisa development1

### PR DESCRIPTION
**Pull request by ojinsi**
Website changes

### CODE REVIEW

File: walkthroughs/week-2-server/examples/text-processor/src/main/java/com/google/sps/servlets/TextProcessorServlet.java [hunk 1]
```diff
@@ -28,9 +28,9 @@
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
Comment (fewtrell) [inline walkthroughs/week-2-server/examples/text-processor/src/main/java/com/google/sps/servlets/TextProcessorServlet.java:29]: Not your fault but could you add a comment that describes what this method does?

File: walkthroughs/week-3-libraries/charts/examples/bigfoot-sightings/src/main/java/com/google/sps/servlets/BigfootDataServlet.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import java.util.LinkedHashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Returns bigfoot data as a JSON object, e.g. {"2017": 52, "2018": 34}] */
+@WebServlet("/bigfoot-data")
+public class BigfootDataServlet extends HttpServlet {
```
Comment (fewtrell) [inline walkthroughs/week-3-libraries/charts/examples/bigfoot-sightings/src/main/java/com/google/sps/servlets/BigfootDataServlet.java]: curious did you write this or is it part of a sample?

File: walkthroughs/week-3-libraries/charts/examples/favorite-colors/src/main/webapp/script.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Google LLC
```
Comment (fewtrell) [inline walkthroughs/week-3-libraries/charts/examples/favorite-colors/src/main/webapp/script.js:1]: I'm assuming the examples (like bigfoot, favorite colors) are unmodified by you.

**Review by fewtrell — APPROVED** [review]

File: walkthroughs/week-0-setup/github-setup-walkthrough.md [hunk 4]
```diff
@@ -98,9 +98,10 @@ needs access to your repo.
 To give your advisor access to your repo, add them as a collaborator.
 
 1.  Navigate to your repo's page in GitHub.
-2.  Go to the `Settings` tab.
-3.  Select `Collaborators`.
-4.  Add your Project Advisor using their GitHub username or email address.
+1.  Go to the `Settings` tab.
+1.  Select `Collaborators`. (*Note*: If you don't see this, 
+    look for `Manage Access` -> `Invite a Collaborator` instead)
+1.  Add your Project Advisor using their GitHub username or email address.
```
Comment (andreaowu) [inline walkthroughs/week-0-setup/github-setup-walkthrough.md]: Why did you change this?

File: walkthroughs/week-2-server/examples/text-processor/src/main/java/com/google/sps/servlets/TextProcessorServlet.java [hunk 5]
```diff
@@ -28,9 +28,9 @@
...
     // Get the input from the form.
-    String text = request.getParameter("text-input");
-    boolean upperCase = Boolean.valueOf(request.getParameter("upper-case"));
-    boolean sort = Boolean.valueOf(request.getParameter("sort"));
+    String text = getParameter(request, "text-input", "");
+    boolean upperCase = Boolean.parseBoolean(getParameter(request, "upper-case", "false"));
+    boolean sort = Boolean.parseBoolean(getParameter(request, "sort", "false"));
```
Comment (andreaowu) [inline walkthroughs/week-2-server/examples/text-processor/src/main/java/com/google/sps/servlets/TextProcessorServlet.java:33]: According to Java naming conventions (#14 on https://www.cs.bgu.ac.il/~majeek/presentations/JavaProgrammingStyle%20Guidelines.html), booleans should be named as "isUpperCase", "isSorted". 

As a side note, this follows in a lot of other languages and makes more clear when you use that variable later in the code that it is a boolean.

**Review by andreaowu — COMMENTED** [review]
Thanks for starting early, good work! Also, thanks for catching grammar / spelling mistakes :)

File: walkthroughs/week-2-server/examples/text-processor/src/main/java/com/google/sps/servlets/TextProcessorServlet.java (hunk 1 above)
Comment (ojinsi) [reply walkthroughs/week-2-server/examples/text-processor/src/main/java/com/google/sps/servlets/TextProcessorServlet.java:29]: I actually didn't write this code, it is the code that came when I downloaded all the files given to us! I'm not actually sure what is does.

File: walkthroughs/week-3-libraries/charts/examples/favorite-colors/src/main/webapp/script.js (hunk 3 above)
Comment (ojinsi) [reply walkthroughs/week-3-libraries/charts/examples/favorite-colors/src/main/webapp/script.js:1]: Yup, didn't modify any examples

File: walkthroughs/week-0-setup/github-setup-walkthrough.md (hunk 4 above)
Comment (ojinsi) [reply walkthroughs/week-0-setup/github-setup-walkthrough.md]: I didn't, I'm not sure why it says I did.

File: walkthroughs/week-2-server/examples/text-processor/src/main/java/com/google/sps/servlets/TextProcessorServlet.java (hunk 5 above)
Comment (ojinsi) [reply walkthroughs/week-2-server/examples/text-processor/src/main/java/com/google/sps/servlets/TextProcessorServlet.java:33]: I didn't write this code, this code came with the downloaded files

File: walkthroughs/week-0-setup/github-setup-walkthrough.md (hunk 4 above)
Comment (andreaowu) [reply walkthroughs/week-0-setup/github-setup-walkthrough.md]: Hmm...that's weird. It shouldn't all say "1." Do you mind fixing that?

Comment (andreaowu) [reply walkthroughs/week-0-setup/github-setup-walkthrough.md]: Hmm...that's weird. It shouldn't all say "1." Do you mind fixing that?

Comment (fewtrell) [reply walkthroughs/week-0-setup/github-setup-walkthrough.md]: markdown automatically converts any digits into sequential numbered lists.  One short cut is  write '1.' for every item. then when you reorder them in code you don't have to change them.  display order will allways be 1,2,3.

from https://meta.stackexchange.com/questions/137077/cant-start-a-numbered-list-on-a-number-other-than-1

"The number that starts a numbered list in the Markdown source will also be the starting number in the rendered version. The actual values of subsequent numbers will be irrelevant as always (the rendered version will just increment one-by-one); only the first number counts."

**Review by andreaowu — APPROVED** [review]
Oops, sorry for the late approval!
